pkg/launcher/uninstall/uninstall.go

```diff
@@ -0,0 +1,71 @@
+package uninstall
```

Comment:
should we call this "disable" ?

Reply:
TBH I don't feel strongly. I think we're likely to iterate and shift and we figure out all the parts.